### PR TITLE
Update Performance documentation.

### PR DESCRIPTION
**Pull request by jeremyjiang-dev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,6 +254,7 @@ pod 'Firebase/Database'      # No watchOS support yet
 pod 'Firebase/Firestore'     # No watchOS support yet
 pod 'Firebase/Functions'     # No watchOS support yet
 pod 'Firebase/Messaging'
+pod 'Firebase/Performance'   # No watchOS and macOS support yet
```
Comment (paulb777) [inline README.md:257]: Might want to hold off on this change until tvOS releases.

**Review by paulb777 — APPROVED** [review]
Thanks!

File: FirebasePerformance/Tests/TestApp/README.md [hunk 2]
```diff
@@ -3,9 +3,13 @@
 ## Setup
 
 Download the `GoogleService-Info.plist` file from [Firebase Console](https://console.firebase.google.com/)
-(for whatever Firebase project you have or want to integrate the `dev-app`). For Autopush environment, store the
+(for whatever Firebase project you have or want to integrate the `dev-app`).
```
Comment (visumickey) [inline FirebasePerformance/Tests/TestApp/README.md:6]: Lets qualify the importance of bundleId matching. I think we can mention the bundle ID here.

File: FirebasePerformance/Tests/TestApp/README.md [hunk 3]
```diff
@@ -3,9 +3,13 @@
...
+
+For Prod environment, store the `GoogleService-Info.plist` under [Plists/Prod/FIRPerfTestApp/](./Plists/Prod/FIRPerfTestApp/). This should be sufficient for most scenarios.
+
+For Autopush environment, store the
```
Comment (visumickey) [inline FirebasePerformance/Tests/TestApp/README.md:10]: Lets qualify that the events generated for the Autopush environment will not be available on the console outside of Google as these are processed on our staging servers.

File: README.md (hunk 1 above)
Comment (visumickey) [reply README.md:257]: Not sure if I understand this right Paul. What is blocking this change to get it?

Comment (paulb777) [reply README.md:257]: There are still open questions on #7174 and even after it merges, this documentation won't be accurate until the next release.

File: .gitignore [hunk 4]
```diff
@@ -35,6 +35,9 @@ FirebaseSegmentation/Tests/Sample/GoogleService-Info.plist
 # FirebaseMLModelDownloader integration tests GoogleService-Info.plist
 FirebaseMLModelDownloader/Tests/Integration/Resources/GoogleService-Info.plist
 
+# FirebasePerformance dev test App and integration tests GoogleService-Info.plist
+FirebasePerformance/**/GoogleService-Info.plist
```
Comment (zijianjoy) [inline .gitignore:39]: Should it be in another PR?

File: FirebasePerformance/README.md [hunk 5]
```diff
@@ -1,10 +1,9 @@
 # Firebase Performance
 
-Firebase Performance is a free mobile app performance analytics service. It
+[Firebase Performance](https://firebase.google.com/docs/perf-mon) is a free mobile app performance analytics service. It
```
Comment (zijianjoy) [inline FirebasePerformance/README.md:3]: `Firebase Performance Monitoring` is the full product name (I previously received tickets from developers who thought that Fireperf is the performance about Firestore)

File: README.md [hunk 6]
```diff
@@ -188,6 +189,12 @@ To run against a production instance, provide a valid GoogleServices-Info.plist
 [public](https://firebase.google.com/docs/database/security/quickstart) while your tests are
 running.
 
+### Firebase Performance
+If you're doing specific Firebase Performance development, see
+[the Performance README](FirebasePerformance/README.md) for instructions about building the SDK
```
Comment (zijianjoy) [inline README.md:194]: We can use the full name `Firebase Performance Monitoring` here and the rest of doc.

File: README.md (hunk 1 above)
Comment (jeremyjiang-dev) [reply README.md:257]: Sounds good, Paul! I can change it to `# No macOS, tvOS or watchOS support yet`. After the tvOS version is released, I can add tvOS support to the doc. WDYT?

Comment (paulb777) [reply README.md:257]: Catalyst support is pending the resolution of the open questions at #7170 and whether it's still ok to claim support until the test failures are investigated and perhaps a similar console validation to what Visu is doing for tvOS.

File: .gitignore (hunk 4 above)
Comment (jeremyjiang-dev) [reply .gitignore]: Yes. This is gone now after I merged it with master.

File: README.md (hunk 1 above)
Comment (jeremyjiang-dev) [reply README.md:257]: Thank you letting me know. I will wait till #7174 and #7170 then.

File: README.md (hunk 6 above)
Comment (jeremyjiang-dev) [reply README.md:194]: Fixed.

File: FirebasePerformance/Tests/TestApp/README.md (hunk 3 above)
Comment (jeremyjiang-dev) [reply FirebasePerformance/Tests/TestApp/README.md]: Added.

File: FirebasePerformance/Tests/TestApp/README.md (hunk 2 above)
Comment (jeremyjiang-dev) [reply FirebasePerformance/Tests/TestApp/README.md]: Great suggestions! Added the bundle IDs there.

File: FirebasePerformance/README.md (hunk 5 above)
Comment (jeremyjiang-dev) [reply FirebasePerformance/README.md]: Sounds good! Fixed.

File: README.md (hunk 1 above)
Comment (jeremyjiang-dev) [reply README.md]: Discussed with the working group, we would like to claim no support for catalyst and tvOS until we fully test and release them. This is not a blocker for this PR. I can update the README after we officially support these platforms.

**Review by zijianjoy — APPROVED** [review]

**Review by visumickey — APPROVED** [review]

**Pull request merged**